Jenkinsfile

```diff
@@ -23,14 +23,14 @@ def jobProperties = [
         rateLimit,
 ]
 // Don't archive sysroot/disk image/kernel images for pull requests and non-default branches:
-def archiveBranches = ['master', 'dev']
+def archiveBranches = ['main', 'master', 'dev']
```

Comment:
This only controls archiving, not which branches we run jobs on, so some other Jenkins config will need updating.  Eventually we could remove master from here, but it's not really urgent as presumably we will cut Jenkins jobs over from one branch to the other.  We probably have to wait for this to land in master/main before we can start cutting Jenkins over however.